ow_plexil/src/plexil-adapter/OwlatInterface.cpp

```diff
@@ -164,7 +164,7 @@ void OwlatInterface::initialize()
     m_subscribers.push_back
       (make_unique<ros::Subscriber>
        (m_genericNodeHandle ->
-        subscribe("/owl_msgs/pan_tilt_position", qsize,
+        subscribe("/pan_tilt_position", qsize,
```

Comment:
Not really in the scope of this ticket, but a small correction to an already merged PR.